pypower/cpf_callbacks.py

```diff
@@ -0,0 +1,49 @@
+"""Callback functions for CPF
+"""
+
+from numpy import c_, amax
+
+
+def cpf_default_callback(k, V_c, lam_c, V_p, lam_p, cb_data, cb_state, cb_args, results=None, is_final=False):
+    """Default callback function for CPF
+    """
+
+    # initialize plotting options
+    pass
```

Comment:
If needed, I'll update these blank parts soon. But it must be added new dependencie like matplotlib. For my own research, that is no need.

Reply:
You could always nest the import of matplotlib inside the function. I don't think it is recommended practice, but it would make matplotlib an optional dependency. Alternatively, the import of matplotlib could be placed inside a try clause that would catch the `ImportError`. An error could be thrown if the user then tries to use the plotting feature.